lib/server/limiter.js

```diff
@@ -2,20 +2,21 @@
 
 const log = require('../logger');
 const errors = require('storj-service-error-types');
-
 const tmpl = '{"rate_limited": {"url": "%s", "method": "%s", "ip": "%s"}}';
 
-module.exports.DEFAULTS = {
-  lookup: function(req) {
-    return [req.headers['x-forwarded-for'] || req.connection.remoteAddress];
-  },
-  onRateLimited: function(req, res, next) {
-    log.info(tmpl,
-             req.originalUrl,
-             req.method,
-             req.headers['x-forwarded-for'] || req.connection.remoteAddress);
-    return next(new errors.RateLimited('Too Many Requests'));
-  },
-  total: 1000,
-  expire: 1000 * 60 // 1 minutes
+module.exports.DEFAULTS = (total=1000, expire=1000*60) => {
+  return {
+    lookup: function(req) {
+      return [req.headers['x-forwarded-for'] || req.connection.remoteAddress, req.route.path];
```

Comment:
Cool, looks like `req.route.path` is different from `req.path` and shows w/o variables filled:
```
/frames/:frame
/reports/exchanges
/buckets/:id/files
/buckets/:id
/buckets/:id/files/:file
/buckets/:id/files/:file/info
/buckets
```